src/dataMap/metaManager/metaSchema.js

```diff
@@ -286,9 +286,11 @@ export default () => {
      * Defines column widths in pixels. Accepts number, string (that will be converted to a number), array of numbers
      * (if you want to define column width separately for each column) or a function (if you want to set column width
      * dynamically on each render).
+     * 
+     * Note that this option will disable {@link AutoColumnSize} plugin.
```

Comment:
We could go further and directly give the tips of the `columns` property for someone (like me) who would like to specify some column width and let the others be autosized.

Reply:
We plan to extend `how-to's` section in our documentation - then we will add inline references to the proper page.